CodenNerd/TeamWork
# #[8] Admin create employee account

#### What does this PR do?
Creates POST route for admin user to create a new employee account
#### Description of Task to be completed?
Admin should be able to create new employee account
This should return the new userId and token
Auth middleware should authenticate who is making the request
#### How should this be manually tested?
Run `npm start` in the terminal to start the app. This runs `nodemon` and then executes `babel-node server.js`
In Postman, go to `localhost:3000/api/v1/auth/create-user` using the POST method. Ensure, the request body contains necessary data
#### What are the relevant Git Projects stories?
#8   Admin can create an employee user account
#### Relevant Images
![Screenshot (82)](https://user-images.githubusercontent.com/34954722/68235904-673ad700-0004-11ea-9fec-d4f15aca8481.png)
